snap/snapcraft.yaml

```diff
@@ -71,46 +71,49 @@
                 build-packages:
                 - qtbase5-dev
                 - dpkg-dev
+                - libgtk-3-dev
                 make-parameters:
                 - FLAVOR=qt5
                 plugin:           make
                 source:           https://github.com/ubuntu/snapcraft-desktop-helpers.git
                 source-subdir:    qt
                 stage-packages:
                 - libxkbcommon0
-                # - ttf-ubuntu-font-family
-                # - dmz-cursor-theme
-                # - light-themes
-                # - adwaita-icon-theme
-                # - gnome-themes-standard
+                # - qt5-gtk-platformtheme
```

Comment:
This can be removed once launchpad launches base 18 build support for snap packages to fix Fall Back UI.